designTokens/shadow.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable sort-keys */
+const _color1 = 'rgba(141,141,141,.05)';
+const _color2 = 'rgba(141,141,141,.06)';
+const _color3 = 'rgba(141,141,141,.1)';
+const _color4 = 'rgba(141,141,141,.2)';
+
+const shadowObject = (x1, y1, blur1, spread1, x2, y2, blur2, spread2, color1, color2, color3, color4) => ({
+  soft: {
+    shadow: {
+      1: {
+        color: {
+          value: color1
+        }
+      },
+      2: {
+        color: {
+          value: color2
+        }
+      }
+    }
+  },
+  hard: {
+    shadow: {
+      1: {
+        color: {
+          value: color3
+        }
+      },
+      2: {
+        color: {
+          value: color4
+        }
+      }
+    }
+  },
+  shadow: {
```

Comment:
Just a thought about the consistency of the output.

Right now we get, e.g.:
$shadow-elevation-level-1-soft-shadow-1-color: rgba(141,141,141,.05);
$shadow-elevation-level-1-soft-shadow-2-color: rgba(141,141,141,.06);
$shadow-elevation-level-1-hard-shadow-1-color: rgba(141,141,141,.1);
$shadow-elevation-level-1-hard-shadow-2-color: rgba(141,141,141,.2);
$shadow-elevation-level-1-shadow-1-offset-x: 0;
$shadow-elevation-level-1-shadow-1-offset-y: 1;
$shadow-elevation-level-1-shadow-1-blur: 8;
$shadow-elevation-level-1-shadow-1-spread: 0;
$shadow-elevation-level-1-shadow-2-offset-x: 0;
$shadow-elevation-level-1-shadow-2-offset-y: 1;
$shadow-elevation-level-1-shadow-2-blur: 2;
$shadow-elevation-level-1-shadow-2-spread: 0;

What do you think about changing the output to:
$shadow-elevation-level-1-shadow-soft-1-color: rgba(141,141,141,.05);
$shadow-elevation-level-1-shadow-soft-2-color: rgba(141,141,141,.06);
$shadow-elevation-level-1-shadow-hard-1-color: rgba(141,141,141,.1);
$shadow-elevation-level-1-shadow-hard-2-color: rgba(141,141,141,.2);